app/assets/stylesheets/layout/common/_contact_panels.scss

```diff
@@ -11,6 +12,21 @@
 .l-contact-panels--homepage {
   border-top: 0;
   margin-top: 0;
+
+  @include respond-to($mq-l) {
+    &:before, &:after {
+      content: "";
+      position: absolute;
+      border-right: 1px solid $color-newsletter-input-border;
+      height: 0;
+      height: 400px;
```

Comment:
Good spot, I've removed this line now. It's unnecessary, since the before / after elements are only shown within the media query anyway.